cmd/operator/kodata/knative-serving/0.15.1/5-serving-storage-version-migration.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 The Knative Authors
```

Comment:
What's the difference between this and 3-serving-storage-version-migration?

Reply:
I do not see there is a `3-serving-storage-version-migration`.